### PR TITLE
[iOS] Fixes Xcode warnings.

### PR DESCRIPTION
**Pull request by jahorton**
Turns out we had a whole host of warnings due to return values not being utilized.  Aside from that, #402 accidentally departed a bit from Xcode's recommended settings, which is reverted here.

Result:  It's nice to have a clean build log.

### CODE REVIEW

File: ios/.gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.pbxproj merge=union
```
Comment (jahorton) [inline ios/.gitattributes]: See http://roadfiresoftware.com/2015/09/automatically-resolving-git-merge-conflicts-in-xcodes-project-pbxproj/ for reference.

Comment (gabrielwong) [inline ios/.gitattributes]: I saw some StackOverflow posts advising against using `merge=union`. Also when rebasing some of the refactor branches, I had to manually merge project files which would have failed with `merge=union` (particularly when files are renamed, reordered or deleted). So I'd prefer to be forced to manually review a few extra merges than have occasional bad merged.

File: ios/keymanios.xcworkspace/xcshareddata/xcschemes/KME-lib.xcscheme [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0830"
```
Comment (gabrielwong) [inline ios/keymanios.xcworkspace/xcshareddata/xcschemes/KME-lib.xcscheme]: Actually please delete this file since it's pointing to an old target.

**Comment by gabrielwong** [comment]
#405 tracks the unused function result warnings and aiming to fix those while refactoring `Manager`.

**Comment by jahorton** [comment]
We're going a different direction with this; see issue #405 and work that addresses it.

**Pull request closed without merging**